tests/e2e/support/commands.js

```diff
@@ -140,20 +129,20 @@ Cypress.Commands.add('setupGameAsSpectator', () => {
 
     cy.url().should('include', '/spectate/');
     cy.window()
-    .its('cuttle.app.config.globalProperties.$store.state.game')
-    .then((gameState) => {
-      expect(gameState.id).to.not.eq(null);
-    });
+      .its('cuttle.app.config.globalProperties.$store.state.game')
+      .then((gameState) => {
+        expect(gameState.id).to.not.eq(null);
+      });
   });
 });
 
-Cypress.Commands.add('signupOpponent', (username, password) => {
+Cypress.Commands.add('signupOpponent', (opponent) => {
```

Comment:
One trick we can bring to bear here is to destructure ([docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions#function_parameters)) the `opponent` parameter here in order to clarify that it's expected to have both a username and password attribute. This would also let us use the shorthand syntax down below. This isn't necessary, but may be cleaner. What do you think?
```suggestion
Cypress.Commands.add('signupOpponent', ({ username, password }) => {
```